Models/NHES/Media/TwoPhaseMixtures/LithiumBromideWater/LiBrW_Utilities/package.mo

```diff
@@ -0,0 +1,212 @@
+within NHES.Media.TwoPhaseMixtures.LithiumBromideWater;
+package LiBrW_Utilities
+    extends Modelica.Icons.UtilitiesPackage;
+    import IFU = Modelica.Media.Water.IF97_Utilities;
+    import IGW = Modelica.Media.IdealGases.Common;
+
+
+
+
+
+
+
+
+    //   function T_phX_der "Derivative function of T_phX"
```

Comment:
Why does this file have a lot of commented material?

Reply:
In attempts to get the media package to work properly, I was unsure which property derivatives had to be given explicitly and which could be handled by Dymola. I will remove the commented functions. If they are needed later, it is not difficult to add them.